components/Cases/CaseLink.tsx

```diff
@@ -32,7 +32,7 @@ const getLink = (
     case 'API_WarningNote':
       return `/people/${caseFormData.mosaic_id}/warning-notes/${
         (caseFormData as WarningNoteCaseFormData).warning_note_id
-      }/view`;
+      }/view?form_name=${caseFormData.form_name}`;
```

Comment:
It'd be good if you could take me through this – I'm not sure the reason for passing the form name as a query param here?